binance-smart-chain-rpc/src/constructors/ethereum/compiler.js

```diff
@@ -1,154 +1,184 @@
+const apisErc20TemplateText = fs.readFileSync(
+ path.resolve(__dirname, "./contracts/APIS_ERC20.sol"),
+ "utf8"
+);
 const apisErc20Template = handlebars.compile(apisErc20TemplateText);
-const apisErc20Abi = fs.readFileSync(path.resolve(__dirname, './abi/APIS_ERC20.json'), 'utf8')
-const apisErc20Compiled = fs.readFileSync(path.resolve(__dirname, './bytecode/APIS_ERC20.json'), 'utf8')
+const apisErc721TemplateText = fs.readFileSync(
+ path.resolve(__dirname, "./contracts/APIS_ERC721.sol"),
+ "utf8"
+);
+const apisErc721Template = handlebars.compile(apisErc721TemplateText);
+const apisErc20Abi = fs.readFileSync(
+ path.resolve(__dirname, "./abi/APIS_ERC20.json"),
+ "utf8"
+);
+const apisErc20Compiled = fs.readFileSync(
+ path.resolve(__dirname, "./bytecode/APIS_ERC20.json"),
+ "utf8"
+);
 
-const createInput = ({
-  templateOptions: { token },
-  type,
-}) => {
-  return {
-    language: 'Solidity',
-    sources: {
-      [`${token.name}.sol`]: {
-        content: type === 'APIS_ERC20' ? apisErc20Template({ token }) : erc20Template({ token }),
-      },
-    },
-    settings: {
-      outputSelection: {
-        '*': {
-          '*': ['*'],
-        },
-      },
+const createInput = ({ templateOptions: { token }, type }) => {
+ const content =
+  type === "APIS_ERC20"
+   ? apisErc20Template({ token })
+   : apisErc721Template({ token });
+ return {
+  language: "Solidity",
+  sources: {
+   [`${token.name}.sol`]: {
+    content,
+   },
+  },
+  settings: {
+   outputSelection: {
+    "*": {
+     "*": ["*"],
     },
-  }
-}
-
+   },
+  },
+ };
+};
 
 function findImports(importFile) {
-  let contents
+ let contents;
+ try {
+  contents = fs.readFileSync(`contracts/${importFile}`, "utf8");
+  return { contents };
+ } catch (error) {
   try {
-    contents = fs.readFileSync(`contracts/${importFile}`, 'utf8')
-    return { contents }
+   contents = fs.readFileSync(
+    path.resolve(__dirname, `../../node_modules/${importFile}`),
+    "utf8"
+   );
+   return { contents };
   } catch (error) {
-      try {
-        contents = fs.readFileSync(path.resolve(__dirname, `../../node_modules/${importFile}`), 'utf8')
-        return { contents }
-      } catch (error) {
-        console.log(error.message)
-        return { error: 'File not found' }
-      }
+   console.log(error.message);
+   return { error: "File not found" };
   }
-
+ }
 }
-
 const compile = ({ input }) => {
-  // New syntax (supported from 0.5.12, mandatory from 0.6.0)
-  return JSON.parse(
-    solc.compile(JSON.stringify(input), { import: findImports })
-  )
-}
-
-
+ // New syntax (supported from 0.5.12, mandatory from 0.6.0)
+ return JSON.parse(
+  solc.compile(JSON.stringify(input), { import: findImports })
+ );
+};
 async function deployContractInternal({
-  contractJson,
-  sender,
-  privateKey,
-  sendOptions,
-  deployArgs = [],
+ contractJson,
+ sender,
+ privateKey,
+ sendOptions,
+ deployArgs = [],
```

Comment:
Let's install eslint when we are bored or if we have some gap in tasks. I've skipped it as a single developer, but it's better practice to have more uniform code formatting.

Reply:
Cool, Right now I am using prettier with 1 space for formatting code  . Would like to know to use eslint

Reply:
Oh! I see -- let's change prettier to 2 spaces